diff --git a/dist/assets/utils/hash.js b/dist/assets/utils/hash.js
index 171f39d58241d3dad4f32ab630cd9c6d4855f3fc..c9ca2c0a45d0f9036018fb0ae6ec7210e3d7b196 100644
--- a/dist/assets/utils/hash.js
+++ b/dist/assets/utils/hash.js
@@ -36,7 +36,8 @@ function propsToFilename(filePath, transform, hash) {
   }
   const prefixDirname = isESMImportedImage(transform.src) ? dirname(filePath) : "";
   let outputExt = transform.format ? `.${transform.format}` : ext;
-  return `${prefixDirname}/${filename}_${hash}${outputExt}`;
+  // Force disable image optimization - return original path without hash and format conversion
+  return `${prefixDirname}/${filename}${ext}`;
 }
 function hashTransform(transform, imageService, propertiesToHash) {
   const hashFields = propertiesToHash.reduce(
diff --git a/dist/core/build/generate.js b/dist/core/build/generate.js
index 84bcbe2182e84d71f533f7aa9c6939b103a2f20b..0ccb63e52c35e0f9780b92673189665b38b2820f 100644
--- a/dist/core/build/generate.js
+++ b/dist/core/build/generate.js
@@ -181,7 +181,8 @@ ${colors.bgGreen(colors.black(` ${verb} static routes `))}`);
     } catch {
     }
   }
-  if (staticImageList.size) {
+  // Force disable image optimization - hardcoded
+  if (false) {
     logger.info("SKIP_FORMAT", `${colors.bgGreen(colors.black(` generating optimized images `))}`);
     const totalCount = Array.from(staticImageList.values()).map((x) => x.transforms.size).reduce((a, b) => a + b, 0);
     const cpuCount = os.cpus().length;
